Sources/__SwiftOnServer_org/Documentation.docc/2024/03/26-getting-started-with-structured-concurrency-in-swift/26-getting-started-with-structured-concurrency-in-swift.md

```diff
@@ -547,10 +530,10 @@ taskGroup.addTask(executorPreference: executor) {
 }
 ```
 
-As you may notice, the `EventLoopExecutor` type is _manually_ retained and released. This is becasue the `addTask` method does not retain the executor. If the EventLoopExecutor type is not retained elsewhere, it will be deallocated before the task is done running, causing a crash.
+As you may notice, the `EventLoopExecutor` type is _manually_ retained and released. This is becasue the ``TaskGroup.addTask`` method does not retain the executor. If the EventLoopExecutor type is not retained elsewhere, it will be deallocated before the task is done running, causing a crash.
```

Comment:
``TaskGroup.addTask(priority:operation:)``

in general, if the method has parameters, you must write the full name of the method for the link to work

Reply:
Doesn't it need to be ``TaskGroup/addTask(priority:operation:)`` ?

Reply:
it allows you to change the rendering behavior :)

see: https://swiftinit.org/docs/swift-unidoc/guides/codelinks#Path%20separator